### DISCUSSION TITLE
Xeon E5 v2

### DISCUSSION THREAD

**Discussion by mmol67** [immich-app/immich#607] [General]
Hello!

My cpu is a Xeon E5-2658 v2, which supports AVX, but I think it doesn't have AVX2. Could it be a problem for Tensorflow? I've read that old cpus cannot run it....

**Comment by bertmelis** [comment]
Just try it and if it doesn't run, build your own ML image: https://github.com/immich-app/immich/discussions/300

**Comment by mmol67** [comment]
Thank you. Tried it using docker compose and tensorflow container starts without error, but when I first added some photos it says

`
tensorflow/core/framework/cpu_allocator_impl.cc:82] Allocation of 33713280 exceeds 10% of free system memory.`

free -h

```
               total        used        free      shared  buff/cache   available
Mem:            15Gi       8,0Gi       187Mi        48Mi       7,3Gi       7,4Gi
Swap:             0B          0B          0B
```
nvidia-smi
` 105MiB /   979MiB`

This is a Xeon E5-2658v3 , 16GB , NVIDIA GTX 750 1GB

Any clue?

**Reply by alextran1502** [reply]
It is a normal log from the Tensorflow library. Nothing to worry about

**Reply by mmol67** [reply]
Thank you!
